amanuensis/resources/userdatamodel/userdatamodel_state.py

```diff
@@ -1,9 +1,16 @@
+from logging import getLogger
 from sqlalchemy import func
-
+from amanuensis.resources.aws import boto_manager
+from amanuensis.auth.auth import current_user
```

Comment:
The model files should only run DB or write/read operation they should use no direct app knowledge. the user should be passed in the function params.